polkadot/runtime/kusama/src/lib.rs

```diff
@@ -279,7 +279,7 @@ impl pallet_babe::Config for Runtime {
 	type WeightInfo = ();
 
 	type MaxAuthorities = MaxAuthorities;
-	type MaxNominators = MaxNominatorRewardedPerValidator;
+	type MaxNominators = MaxExposurePageSize;
```

Comment:
https://github.com/paritytech/substrate/pull/13498/files/29f800bf4eaa0d4d9a837591461e7ee6e438679f..ec0694bfb69c5625ba6d2fe76da8e0446dcd71d7#r1299430753

https://github.com/paritytech-secops/srlabs_findings/issues/2#issuecomment-1685860874

Reply:
Isn't this a pretty bad name? I belive what `pallet_babe` wants to convey here is `MaxNominatorsPerValidator`?

Reply:
Also, indeed, the correct value for this should be something like `max_pages * page_size`, but I trust that this will come in a follow-up.

Reply:
Its terrible but its just more apparent now :(. `MaxNominatorsPerValidator ` was also wrong and exactly same thing.

Reply:
I will add a comment about this in the code. Not sure what else I can do in the scope of current PR.